scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,442 @@
+# -*- coding: utf-8 -*-
+"""
+This script computes four myelin indices maps from the Magnetization Transfer
+(MT) and inhomogeneous Magnetization Transfer (ihMT) images. Magnetization
+Transfer is a contrast mechanism in tissue resulting from the proton exchange
+between non-aqueous protons (from macromolecules and their closely associated
+water molecules, the "bound" pool) and protons in the free water pool called
+aqueous protons. This exchange attenuates the MRI signal, introducing
+microstructure-dependent contrast. MT's effect reflects the relative density
+of macromolecules such as proteins and lipids, it has been associated with
+myelin content in white matter of the brain.
+
+Different contrasts can be done with an off-resonance pulse prior to image
+acquisition (a prepulse), saturating the protons on non-aqueous molecules,
+by applying different frequency irradiation. The two MT maps and two ihMT maps
+are obtained using five contrasts: single frequency positive or negative and
+dual frequency with an alternation of both positive and negative frequency
+(saturated images); and one unsaturated contrast as reference (T1weighted).
+
+The output consist in two types of images:
+        five contrast images and four (ih)MT maps.
+
+Contrast images :
+    - AltNP : alternating negative and positive frequency pulses
+    - altPN : alternating positive and negative frequency pulses
+    - positive : pulses applied at positive frequency
+    - negative : pulses applied at negative frequency
+    - reference : no pulse
```

Comment:
I see from your example that all these images need to be co-registered before running the script. Correct?  Can you add this info to the help please but a recommendation (ANTS? or something in the docs/)